test/e2e/specs/site-editor/writing-flow.spec.js

```diff
@@ -23,8 +23,12 @@ test.describe( 'Site editor writing flow', () => {
 		await admin.visitSiteEditor( {
 			postId: 'emptytheme//header',
 			postType: 'wp_template_part',
+			canvas: 'edit',
 		} );
-		await editor.canvas.click( 'body' );
+
+		// Wait for the template part to be loaded.
+		await page.waitForLoadState( 'networkidle' );
```

Comment:
Wild idea: How about adding this to `visitSiteEditor` so that we solve all the site editor flakiness automatically? 😅

Reply:
Oh, this gives me some ideas. I'll update in https://github.com/WordPress/gutenberg/pull/49374.

Reply:
Updated in [7aff89a](https://github.com/WordPress/gutenberg/pull/49374/commits/7aff89a0322ec0ec94b1d761164ce282f1a57efc)!

Reply:
We can also update `visitSiteEditor` and `visitAdminPage` to accept options passed to `page.goto` - https://playwright.dev/docs/api/class-page#page-goto-option-url.